### ISSUE TITLE
v1 does not authenticate for gsutil cp

### ISSUE THREAD

**Issue by OleksiiKachan** [google-github-actions/auth#241]
### TL;DR

bumped auth and setup actions to v1
I use json service account
right after these actions I run gsutil cp
401 Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object.

works fine with v1

### Expected behavior

should run `gsutil cp` as authenticated caller

### Observed behavior

runs `gsutil cp` as anonymous caller

### Action YAML

```yaml
- name: Authenticate to gcloud
  uses: "google-github-actions/auth@v1"
  with:
    credentials_json: "${{ secrets.GCP_SERVICE_ACCOUNT }}"
- uses: 'google-github-actions/setup-gcloud@v1'
- name: Decrypt
  run: yarn decrypt --${{ env.DEPLOYMENT_ENVIRONMENT }} - this one uses gsutil cp
- name: Deploy
  uses: 'google-github-actions/deploy-appengine@v0'
  with:
    project_id: ${{ secrets.PROJECT_ID }}
    deliverables: ${{ secrets.APP_CONFIG }}
```

**Comment by sethvargo** [comment]
Hi there @OleksiiKachan 👋!

Thank you for opening an issue. Our team will triage this as soon as we can. Please take a moment to review the [troubleshooting steps](https://github.com/google-github-actions/auth/blob/main/docs/TROUBLESHOOTING.md) which lists common error messages and their resolution steps. Having the debug logs would be super helpful, especially if you can get logs as `v0` vs `v1`.

**Comment by Otterverse** [comment]
I ran into the same problem. It's not logging in an account. `gcloud auth list` returns empty, and `gcloud info` shows no account set. I fixed it in my workflows by adding a step after auth and gcloud setup that looks like this:

```
    - name: Fix Auth
      run: yes | gcloud auth login --cred-file="$GOOGLE_APPLICATION_CREDENTIALS"
```

**Comment by sethvargo** [comment]
Hi @Otterverse 

I believe we fixed this in setup-gcloud with 1.0.1 - could you please confirm?

**Comment by OleksiiKachan** [comment]
@sethvargo works for me now. thanks
